supplemental-docs/AWS_LAMBDA.md

```diff
@@ -19,6 +20,112 @@ exports.handler = function (event) {
 };
 ```
 
+## Creating an AWS Lambda layer
+
+With layers, you can create a shared set of dependencies for your AWS Lambda functions that can be imported
+as if they were included in the functions themselves. 
+
+The benefits of doing this include finer control over the version of the AWS SDK you are using, and not
+having to upload the AWS SDK as part of your individual functions' code.
+
+This information was taken from https://docs.aws.amazon.com/lambda/latest/dg/nodejs-layers.html.
+
+### Create the content for your Layer
+
+In a clean folder, create a `package.json` with your required dependencies.
+
+```json5
+// package.json
+{
+  "dependencies": {
+    "@aws-sdk/client-s3": "<=3.750.0",
+    "@aws-sdk/client-dynamodb": "<=3.750.0",
+    "@aws-sdk/lib-dynamodb": "<=3.750.0",
+    "@aws-sdk/client-lambda": "<=3.750.0"
+  }
+}
+```
+
+In this example we've used the `<=` version range to ensure the AWS SDK dependencies use the same transitive dependency, 
+i.e., core package versions. Run `npm install` or equivalent to create the `node_modules` folder, and then zip the contents
+into the following structure:
+
+```
+layer_content.zip
+└ nodejs
+  └ node_modules
+    └ @aws-sdk/etc.
+```
+
+### Deploying the AWS Lambda layer to your account
+
+```js
+// example: deploying the AWS Lambda layer.
+import { Lambda } from "@aws-sdk/client-lambda";
+import fs from "node:fs";
+
+const lambda = new Lambda({
+  region: "us-west-2",
+});
```

Comment:
Should we remove the region from the examples altogether to keep the code simple?
```suggestion
const lambda = new Lambda();
```
Ditto on line 95.